engine/src/main/java/com/odysseusinc/arachne/executionengine/execution/r/DockerOverseer.java

```diff
@@ -36,6 +36,8 @@ public class DockerOverseer extends AbstractOverseer {
      */
     private volatile int pos;
     private final DockerClient client;
+    private String containerId;
+    private static final StringBuilder executionLogs = new StringBuilder();
```

Comment:
There is already a field `stdout` that contains all the execution logs. Why do you need to create another one?

Reply:
You are right!.